src/components/switchTheme/useSwitchTheme.tsx

```diff
@@ -0,0 +1,56 @@
+import { createContext, ReactNode, useCallback, useContext, useEffect, useState } from 'react';
+
```

Comment:
the name of file says there is only hook here, and ".tsx" says there is JSX here. Wrong naming.

Reply:
Changed the structure and added file hook.ts for useTheme

Reply:
where?